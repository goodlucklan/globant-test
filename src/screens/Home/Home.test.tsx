import HomeScreen from "./Home.screen";

describe("HomeScreen test", () => {
  test("render a Screen Component", () => {
    expect(<HomeScreen />).toMatchSnapshot();
  });
});
