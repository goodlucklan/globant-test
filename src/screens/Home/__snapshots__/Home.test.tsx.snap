// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen test render a Screen Component 1`] = `<HomeScreen />`;
